picard/ui/theme.py

```diff
@@ -87,7 +87,7 @@ def _missing_(cls, value):
     func=QtGui.QColor(255, 87, 160, 255),  # magenta
     var=QtGui.QColor(252, 187, 81, 255),  # orange
     escape=QtGui.QColor(75, 239, 31, 255),  # green
-    special=QtGui.QColor(255, 87, 160, 255),  # blue
+    special=QtGui.QColor(150, 150, 239),  # blue
     noop=QtGui.QColor(4, 231, 213, 255),  # cyan
```

Comment:
I'm not entirely sure about this change. I find the blueish color hard to read and don't really see why we need to use a separate color for the parentheses and commas. But if we change it, shouldn't we also change it in the light theme? Maybe I also don't fully understand what this intents to fix.

If it is only about highlighting invalid functions I would suggest to introduce a separate "error" color just for this.

Small side note: For consistency I would either remove the alpha from all values or keep it in this value as well.

Reply:
Perhaps you missed the point: it has a different color, but in light mode only. I see no reason to have same color in dark mode, while they differ in light mode.

Reply:
> If it is only about highlighting invalid functions

It was here before, not for invalid functions only.

> I would suggest to introduce a separate "error" color just for this.

Yes, I thought about that too, but first I'd like to make those colors configurable.

Reply:
> Perhaps you missed the point: it has a different color, but in light mode only.

Not it hasn't, it's `QtCore.Qt.GlobalColor.blue` for both function names and the parentheses / commas. That's why this change confused me.

Reply:
Ah damn, I was confused by the comment on https://github.com/metabrainz/picard/blob/e4168b53859ab8f4c2b5fa205f292349770a3d3c/picard/ui/theme.py#L90 (vs https://github.com/metabrainz/picard/blob/e4168b53859ab8f4c2b5fa205f292349770a3d3c/picard/ui/theme.py#L87)

Ok, so we have 2 identical colors, I'll revert this change.

Reply:
Done & rebased

Reply:
I see. I think that's an example of a comment not following code changes. I likely added the comments first before replacing the colors, and as the light theme uses "blue" here the comment was blue. When changing the actual color to magenta only the comment for the first use was updated. So we should fix the comment here, I did so in f78d02e415d89b608735548d8426e5d69dccb3c9